subsys/net/lib/lwm2m/lwm2m_obj_firmware.c

```diff
@@ -143,13 +143,35 @@ static int package_write_cb(u16_t obj_inst_id,
 			    u8_t *data, u16_t data_len,
 			    bool last_block, size_t total_size)
 {
-	SYS_LOG_DBG("PACKAGE WRITE");
+	u8_t state;
+	int ret = 0;
```

Comment:
usually, in net stack, we try to have declaration with assignment as first lines, then the rest. So I would reverse the order here.